packages/spec-parser/src/specParser.browser.ts

```diff
@@ -163,6 +165,22 @@ export class SpecParser {
     throw new Error("Method not implemented.");
   }
 
+  /**
+   * Generates and update artifacts from the OpenAPI specification file. Generate Adaptive Cards, update Teams app manifest, and generate a new OpenAPI specification file.
+   * @param manifestPath A file path of the Teams app manifest file to update.
+   * @param filter An array of strings that represent the filters to apply when generating the artifacts. If filter is empty, it would process nothing.
+   * @param outputSpecPath File path of the new OpenAPI specification file to generate. If not specified or empty, no spec file will be generated.
+   */
+  // eslint-disable-next-line @typescript-eslint/require-await
+  async generateForCopilot(
+    manifestPath: string,
+    filter: string[],
+    outputSpecPath: string,
+    apiPluginFileName: string,
```

Comment:
just want to confirm if you hope to pass the file name. might  be better if it is path?

Reply:
Will update to use path instead of name